### PR TITLE
update docs + add assertion, loggers

### PR DESCRIPTION
**Pull request by ChenKangg**

### CODE REVIEW

File: src/main/java/seedu/omnitravel/ui/Ui.java [hunk 1]
```diff
@@ -80,28 +80,29 @@ public static void helpCommand(){
...
-                "13. untag <activity number>\n" +
-                "14. update <update> <date> <duration> <tag>\n" +
-                "15. findtag <tag name> <exclusion>\n" +
-                "16. findtype <type> <exclusion>\n" +
-                "17. listtags \n" +
-                "18. expense <activity number> <expense amount>\n" +
-                "19. removeexpense <activity number>\n" +
-                "20. totalexpense <type>\n" +
-                "21. change <amount> /from <current currency> /to <changed currency>\n");
+                        "2. help: Get all commands for the chatbot\n" +
```
Comment (ChinYanXu) [inline src/main/java/seedu/omnitravel/ui/Ui.java]: Why did you add these spacing?

**Review by daryltay415 — APPROVED** [review]
